kqueen/engines/aks.py

```diff
@@ -238,10 +241,12 @@ def resize(self, node_count, **kwargs):
         )
 
         try:
-            self.client.managed_clusters.create_or_update(self.resource_group_name, self.cluster.id, managed_cluster)
+            self.client.managed_clusters.create_or_update(self.resource_group_name,
+                                                          self.cluster.id, managed_cluster)
             # TODO: check if resizing response is healthy
         except Exception as e:
-            msg = 'Resizing cluster {} failed with following reason:'.format(self.cluster.id)
+            msg = 'Resizing cluster {} failed with following reason: {}'\
```

Comment:
if you are logging exception e will be logged anyway, we just need to return repr(e) instead of msg here